traffic_ops/testing/api/v3/tc-fixtures.json

```diff
@@ -3368,4 +3368,4 @@
             "description": "bar"
         }
     ]
-}
+}
```

Comment:
Removing this trailing newline makes this not a text file

Reply:
aha, I see.. i'll add it back